2025-05-23 14:50:22.392 | ERROR    | common.request.request_service:request:26 | HTTPConnectionPool(host='openapi.portal.nbpod3-31-181-20030.4a.cmit.cloud', port=80): Max retries exceeded with url: /apisix/plugin/jwt/sign (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x00000268C9695290>, 'Connection to openapi.portal.nbpod3-31-181-20030.4a.cmit.cloud timed out. (connect timeout=None)'))
2025-05-23 15:46:48.957 | ERROR    | common.request.request_service:request:26 | HTTPConnectionPool(host='openapi.portal.nbpod3-31-181-20030.4a.cmit.cloud', port=80): Max retries exceeded with url: /apisix/plugin/jwt/sign (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022EB7796BD0>: Failed to establish a new connection: [WinError 10013] 以一种访问权限不允许的方式做了一个访问套接字的尝试。'))
